### PR TITLE
send to server upon grab release

### PR DESCRIPTION
**Pull request by sethalves**
- fix a bug what caused desynchronization when an AvatarActionHold is delted

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4ed8a1e5d116a663bdf3e0642e21a239548999d1 available at: https://gist.github.com/136c25295be35a95f3d4<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6329.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6329.dmg

**Comment by sethalves** [comment]
test this please

**Comment by AndrewMeadows** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4ed8a1e5d116a663bdf3e0642e21a239548999d1 available at: https://gist.github.com/136c25295be35a95f3d4<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6329.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6329.dmg

**Comment by imgntn** [comment]
tested this on several toybox scripts and the objects seem to behave as expected

File: libraries/entities/src/EntityTree.cpp [hunk 1]
```diff
@@ -661,7 +671,9 @@ int EntityTree::processEditPacketData(NLPacket& packet, const unsigned char* edi
                         qCDebug(entities) << "   properties:" << properties;
                     }
                     if (wantTerseEditLogging()) {
-                        qCDebug(entities) << "edit" << entityItemID.toString() << properties.listChangedProperties();
+                        QList<QString> changedProperties = properties.listChangedProperties();
+                        fixupTerseEditLogging(properties, changedProperties);
+                        qCDebug(entities) << "edit" << entityItemID.toString() << changedProperties;
```
Comment (ZappoMan) [inline libraries/entities/src/EntityTree.cpp]: note: you don't have to "toString" and EntityItemID it already has a streaming operator to qDebug.

File: libraries/entities/src/EntityTree.cpp [hunk 2]
```diff
@@ -611,6 +611,16 @@ EntityItemPointer EntityTree::findEntityByEntityItemID(const EntityItemID& entit
     return foundEntity;
 }
 
+void EntityTree::fixupTerseEditLogging(EntityItemProperties& properties, QList<QString>& changedProperties) {
```
Comment (ZappoMan) [inline libraries/entities/src/EntityTree.cpp]: why did you make this part of EntityTree? This looks like just a plain old helper function that doesn't really need any access to the EntityTree internals.

**Pull request merged**